legal-api/src/legal_api/resources/v2/admin/configuration.py

```diff
@@ -28,13 +28,23 @@
 @cross_origin(origin='*')
 @jwt.has_one_of_roles([UserRoles.staff])
 def get_configurations():
-    """Return a list of configurations."""
-    configurations = Configuration.all()
-    return jsonify({
-        'configurations': [
-            configuration.json for configuration in configurations
-        ]
-    }), HTTPStatus.OK
+    """Return a list of configurations, optionally filtered by name."""
+    filter_name = request.args.get('name', None)
+    if filter_name:
+        configuration = Configuration.find_by_name(filter_name)
```

Comment:
I envisioned being able to query potentially more than one configuration entry by name.  This implementation wouldn't support that.

e.g. GET /admin/configurations?names=NUM_DISSOLUTIONS_ALLOWED,MAX_DISSOLUTIONS_ALLOWED

Reply:
Thanks for the feedback! I will update the code to ensure it works as expected.